apps/src/templates/instructions/HelpTab.jsx

```diff
@@ -0,0 +1,59 @@
+import {showVideoDialog} from "../../code-studio/videos";
+import React, {PropTypes} from 'react';
+
+var styles = {
+  referenceArea: {
+    marginTop: 20,
+  },
+  videoLink: {
+    display: 'inline-block',
+    margin: 8,
+    fontWeight: 'bold',
+    fontSize: 16,
+    lineHeight: 25 + 'px',
+  },
+  videoThumbnail: {
+    borderRadius: 5,
+    height: 40,
+    width: 'auto',
+    marginRight: 8
+  }
+};
+
+const HelpTab = React.createClass({
```

Comment:
should also use `class` instead of `createClass` here

Reply:
I might call this something like HelpTabContents. Currently your `Tab` class (as I understand it) refers just to the tab portion above the content area, and is use by both regular instructions and help. This class, though it has Tab in the name, I believe represents only the tab contents.